### ISSUE TITLE
main.py里面loss是不是写错了

### ISSUE THREAD

**Issue by Haitons** [SherlockLiao/Char-RNN-PyTorch#4]
` loss = self.criterion(score, y)`是不是应该写成`loss = criterion(score, y.view(-1))`啊
